### DISCUSSION TITLE
LSM6DSO

### DISCUSSION THREAD

**Discussion by GiuseppeTorre** [stm32duino/Arduino_Core_STM32#2194] [Q&A] [answered]
Hello everyone. How can i set the offset in the LSM6DSO sensor in the ST iks01a3 board

**Comment by fpistm** [comment]
@cparata could you answer, please?

**Comment by cparata** [comment]
Hello @GiuseppeTorre ,
which offset do you mean?
Best Regards,
Carlo

**Reply by GiuseppeTorre** [reply]
a man used this instruction       :       mpu.setYAccelOffset(1593);
  mpu.setZAccelOffset(963);
  mpu.setXGyroOffset(40);              for the MPU-6050 sensor.

**Reply by cparata** [reply, marked as answer]
So, probably you mean an offset to calibrate the sensor. Actually, the calibration of the sensor is out-of-the-scope of the library. Anyway, you can calculate it at application level with the algorithm that you prefer and then you can substract the offset computed to the values returned by the sensor always in the application. But, there are not APIs to manage offset in the library.
Best Regards,
Carlo

**Reply by GiuseppeTorre** [reply]
thanks Carlo. love you
